iFixit/css-gather
# css-gather: Update deps

This just updates a few of the deps for this repo. Nothing crazy I don't think.

QA:
--
Run through the steps of the README on this repo. Make sure it still works.

Connects: https://github.com/iFixit/ifixit/issues/50016
